edb/schema/inheriting.py

```diff
@@ -473,38 +474,38 @@ def _apply_field_ast(
 def delta_bases(
     old_bases: Iterable[str], new_bases: Iterable[str]
 ) -> Tuple[
-    Tuple[so.ObjectRef, ...],
+    Tuple[so.ObjectShell, ...],
     Tuple[
         Tuple[
-            List[so.ObjectRef],
-            Union[str, so.ObjectRef, Tuple[str, so.ObjectRef]],
+            List[so.ObjectShell],
+            Union[str, so.ObjectShell, Tuple[str, so.ObjectShell]],
```

Comment:
Probably should use type aliases to make type signatures like this easier to digest